python/lsst/pipe/tasks/postprocess.py

```diff
@@ -789,6 +791,167 @@ def _makeArgumentParser(cls):
         return parser
 
 
+class ConsolidateVisitSummaryConnections(pipeBase.PipelineTaskConnections,
+                                         dimensions=("instrument", "visit",),
+                                         defaultTemplates={}):
+    calexp = cT.Input(
+        doc="Calibrated exposures used for metadata",
```

Comment:
They're not really "calibrated exposures", since they still have units of counts (and we hope to replace the "calexp" butler term). How about "processed exposures" instead?